serverless-mfa-api/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM golang:1.18
+
+WORKDIR /initDynamo
+COPY ./initDynamo /initDynamo/
+RUN go get ./... && go mod tidy && go build ./...
+
+WORKDIR /src
+
+RUN mkdir override
+RUN mkdir server
+ENV BRANCH=develop
+
+RUN curl -sSfL https://raw.githubusercontent.com/cosmtrek/air/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
+
+# Download the necessary serverless-mfa-api-go files for running the api in a local container
```

Comment:
Would it be better to have our serverless-mfa-api-go repo publish itself as a Docker image, for this sort of test/dev use? That feels like something that would better belong in that repo rather than this one.

That would also remove the need for having any serverless-mfa-api-go code here in this otherwise PHP repo, which could be helpful for PR reviews on this repo (i.e. not requiring the reviewers to understand both Go and PHP).

Reply:
It was my semi-sneaky way of providing you with more go exposure. ;-)

Reply:
😆 Okay, if that's the best reason available, then it _really_ seems better to ~~delete all this Go code~~ (😉) move this into the serverless-mfa-api-go repo, since that really is the right place for serverless-mfa-api-go code.

Reply:
I'll look into using the aws php sdk for initializing dynamodb data.  I had been using just the cli commands in a bash script, but they didn't seem to work right when I tried to add a byte array as the value of an entry.